### ISSUE TITLE
Lang keys containing colons

### ISSUE THREAD

**Issue by priithansen** [martinlindhe/laravel-vue-i18n-generator#84]
Hei

Our lang keys are separated with colons (:) for namespacing reasons and it seems that all keys are also sent through adjustString() which replaces them similar to values. 
I can't think of the benefit of sending keys through the same transformation, am I missing something?

```php
'spa:page:title:dashboard' => 'Dashboard',
```

ends up

```js
 "spa{page}{title}{dashboard}": "Dashboard",
```

**Comment by martinlindhe** [comment]
Colons are used in laravel translations to indicate named parameters, which is translated to {name} which is vue-i18n syntax.

Most people use dot to namespace their translation strings, such as `'spa.page.title.dashboard' => 'Dashboard',` which is also what you'd see in the Laravel manual.

**Comment by priithansen** [comment]
Jeah named parameters make sense but why even parse them from **keys** as there are no parameters there?

As for me it seems it just disables using : in translation keys without any benefit?

**Comment by martinlindhe** [comment]
I have no opinion about what you are arguing for.
If you insist this should be changed, then by all means provide a PR.

**Comment by martinlindhe** [comment]
Current behavior was implemented to fix #64.

**Comment by martinlindhe** [comment]
Current behavior was implemented to fix #64.
